Content.Client/UserInterface/Systems/Chat/Controls/ChatInputBox.cs

```diff
@@ -1,5 +1,9 @@
-﻿using Content.Shared.Chat;
+﻿using Content.Client._NF.Language.Systems.Chat.Controls;
+using Content.Client.UserInterface.Systems.Language;
```

Comment:
should this not be `using Content.Client._NF.Language;`

Reply:
honestly, namespaces are fucked up here, _NF should be gone except in cases where it's absolutely needed (like .xaml.cs files)